src/Board.ts

```diff
@@ -112,3 +120,34 @@ function hasFreeSpotBeforeToMove(
 
   return !positionExistsInBoard(boardGraph, `${hIndex},${vIndex}`);
 }
+
+
+export function moveMarble(
+  board: Board,
+  userMove: { marblePosition: string; marbleDirection: string }
```

Comment:
marbleDirection is a char with only 4 possibilities, Z is not allowed, you can use enum to be more explicit

https://stackoverflow.com/questions/50248807/what-is-the-typescript-equivalent-of-proptypes-oneof-restrict-a-variable-to-s/50248842

Reply:
Alexandre created a Direction type in another development, I will apply it. Do you think it's enough?